cfpb/hmda-platform
# Parse institution data

Closes #774 

Testing instructions:

* Make sure the postgres "institutions" table is removed with `drop table institutions;`
* In the hmda-platform project, run `sbt`, then `clean`, then select the panel project with `project panel`
* Run the panel project with `run /path/to/panelData.csv` (an example test file is on Google Drive, under `HMDA/Panel/inst_data_2016_1_trim.csv`)
* You currently have to force-stop the program when it's finished with control+C (this might get fixed later)
* Restart `sbt`, the select `project api` and `reStart`. (Don't clean!!!)
* Make sure your authorization header is updated with a few real IDs (which you can get from the log output, it's the first field argument for `InstitutionQuery`)
* Try out the endpoint `localhost:8080/institutions`, and you should get a response with real data!
